app/src/main/java/com/mymeetings/android/model/CalendarEventAlert.kt

```diff
@@ -2,12 +2,12 @@ package com.mymeetings.android.model
 
 import com.mymeetings.android.model.strategies.ViewAlertType
 
-data class CalendarEventWithAlert(
-    val calendarEvent: CalendarEvent,
+data class CalendarEventAlert(
```

Comment:
Let's not have a model explosion here, currently we have two models `CalendarEventAlert` and `CalendarEventAlertUIModel`, let's merge them into one and name it `CalendarEventAlert`.

Reply:
No. CalendarEventAlert is different Entity. CalendarEvent is different entity.     Lets change CalendarEventAlertUIModel to different naming convention.

Reply:
The model `CalendarEventAlert` just has two fields which is better moved to the the UI model. Let's not create a hundred different models which is difficult to track. If we need to discuss on this let's do it and proceed.